### PR TITLE
Add SIRIND Brick65

### PR DESCRIPTION
**Pull request by eerraa**
SIRIND Brick65

## Description
Brick65 is a 65% ANSI layout Custom Keyboard
Powered by ATMEGA32U4

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by waffle87 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/era/sirind/brick65/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Brick65
+
+![Brick65](https://i.imgur.com/qyQYWfj.jpg)
```
Comment (fauxpark) [inline keyboards/era/sirind/brick65/readme.md:3]:
```suggestion
![Brick65](https://i.imgur.com/qyQYWfjh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/era/sirind/brick65/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_ESC,     KC_1,     KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS,  KC_EQL,   KC_BSLS,   KC_BSPC,   KC_DEL, 
+        KC_TAB,     KC_Q,     KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,  KC_RBRC,             KC_BSLS,   KC_SCRL, 
+        KC_CAPS,    KC_A,     KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,            KC_ENT,         
+        KC_LSFT,              KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT,             KC_UP, 
+        KC_LCTL,    KC_LWIN,  KC_LALT,                   KC_SPC,           KC_SPC,           KC_SPC,  KC_RALT, KC_RCTL,            KC_LEFT,   KC_DOWN,   KC_RGHT
```
Comment (zvecr) [inline keyboards/era/sirind/brick65/keymaps/default/keymap.c:26]:
```suggestion
        KC_LCTL,    KC_LWIN,  KC_LALT,                   KC_SPC,           MO(1),           KC_SPC,  KC_RALT, KC_RCTL,            KC_LEFT,   KC_DOWN,   KC_RGHT
```

Comment (eerraa) [reply keyboards/era/sirind/brick65/keymaps/default/keymap.c]: Thank you sir!

File: keyboards/era/sirind/brick65/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_ESC,     KC_1,     KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS,  KC_EQL,   KC_BSLS,   KC_BSPC,   KC_DEL, 
+        KC_TAB,     KC_Q,     KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,  KC_RBRC,             KC_BSLS,   KC_SCRL, 
+        KC_CAPS,    KC_A,     KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,            KC_ENT,         
+        KC_LSFT,              KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT,             KC_UP, 
+        KC_LCTL,    KC_LWIN,  KC_LALT,                   KC_SPC,           MO(1),            KC_SPC,,  KC_RALT, KC_RCTL,            KC_LEFT,   KC_DOWN,   KC_RGHT
```
Comment (fauxpark) [inline keyboards/era/sirind/brick65/keymaps/default/keymap.c:26]:
```suggestion
        KC_LCTL,    KC_LWIN,  KC_LALT,                   KC_SPC,           MO(1),            KC_SPC,  KC_RALT, KC_RCTL,            KC_LEFT,   KC_DOWN,   KC_RGHT
```
Typo in last review

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**